dev/TeachingTip/TeachingTip.idl

```diff
@@ -64,6 +64,14 @@ runtimeclass TeachingTipClosingEventArgs
     Windows.Foundation.Deferral GetDeferral();
 };
 
+
+[MUX_PUBLIC]
```

Comment:
I think this would need to be marked as preview API first (@MikeHillberg correct me if I'm wrong). You can find an example of this [here](https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/dev/PagerControl/PagerControl.idl#L13-L20).